subprojects/file-watching/src/integTest/groovy/org/gradle/internal/watch/LoggingFileSystemWatchingIntegrationTest.groovy

```diff
@@ -97,6 +97,36 @@ class LoggingFileSystemWatchingIntegrationTest extends AbstractFileSystemWatchin
         waitForChangesToBePickedUp()
         then:
         daemon.logContains(logLineCountBeforeChange, NativeLogger.name)
+    }
+
+    @Requires(
+        value = IntegTestPreconditions.NotConfigCached,
```

Comment:
🤔 I think this won't cut it. Soon config cache will be the only option, then what? Can we instead reset the log, and ignore the messages produced by CC? Or expect them specifically when CC is enabled?

Reply:
We can do some different assertion for CC. I will ask cc team what they think is best option with such tests

Reply:
Hmm, thinking more about that. I guess the question is, do we want to fix this behaviour are we ok with it?
If we are ok with it, adjusting test for cc might make sense. 
But if not, then I think it might be better to mark is as incompatible with cc.

What do you think?

Reply:
From Rafael:
> Rafael
We will revisit tests marked excluded under the CC-enabled executer at some point in the future, If you can add some context for the next reader, that should be fine

So I think we are fine. I added also a link to the issue in the reason part.